engine/src/flutter/display_list/testing/dl_test_snippets.cc

```diff
@@ -1084,25 +1089,46 @@ std::vector<DisplayListInvocationGroup> CreateAllRenderingOps() {
             },
             1u},
        }},
-      {"DrawTextBlob",
+      {"DrawText",
        {
-           {1, 24, 1,
+           {1, 32, 1,
             [](DlOpReceiver& r) {
-              r.drawTextBlob(GetTestTextBlob(1), 10, 10);
+              r.drawText(DlTextSkia::Make(GetTestTextBlob(1)), 10, 10);
             }},
-           {1, 24, 1,
+           {1, 32, 1,
             [](DlOpReceiver& r) {
-              r.drawTextBlob(GetTestTextBlob(1), 20, 10);
+              r.drawText(DlTextSkia::Make(GetTestTextBlob(1)), 20, 10);
             }},
-           {1, 24, 1,
+           {1, 32, 1,
             [](DlOpReceiver& r) {
-              r.drawTextBlob(GetTestTextBlob(1), 10, 20);
+              r.drawText(DlTextSkia::Make(GetTestTextBlob(1)), 10, 20);
             }},
-           {1, 24, 1,
+           {1, 32, 1,
+            [](DlOpReceiver& r) {
+              r.drawText(DlTextSkia::Make(GetTestTextBlob(2)), 10, 10);
+            }},
+       }},
+#if IMPELLER_SUPPORTS_RENDERING
+      {"DrawText",
```

Comment:
(Still while being `#if`d), move these cases into the previous block so we get ==/!= checks between calls that differ in Skia/Impeller-ism. Also, the fact that both blocks are labeled "DrawText" makes it hard to know which test case failed when it reports "bad thingy in DrawText variant 3".

Reply:
So instead of 2 blocks both named "DrawText", we have 1 block that only has 4 cases for Skia and 8 cases for Skia+Impeller...